session/pingpong/consumer_balance_tracker.go

```diff
@@ -197,6 +197,12 @@ func (cbt *ConsumerBalanceTracker) ForceBalanceUpdate(id identity.Identity) uint
 	}
 
 	grandTotal, err := cbt.consumerGrandTotalsStorage.Get(id, cbt.accountantAddress)
+	if err == ErrNotFound {
+		if err := cbt.recoverGrandTotalPromised(id); err != nil {
+			log.Error().Err(err).Msg("Could not recover Grand Total Promised")
+		}
+		grandTotal, err = cbt.consumerGrandTotalsStorage.Get(id, cbt.accountantAddress)
```

Comment:
What if this one fails too?

Reply:
The error is handled by the next if block. If it's `not found` again we continue assuming `grandTotal` is 0.